addons/partner_autocomplete/__manifest__.py

```diff
@@ -27,4 +27,5 @@
         'static/src/xml/partner_autocomplete.xml',
     ],
     'auto_install': True,
+    'post_init_hook': '_enrich_base_company',
```

Comment:
If this feature targets the SaaS, please take into account that we use pre-built database templates that are duplicated on-demand, so the init hook is likely to run on a template database rather than a valid one.